examples/text_input/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
```

Comment:
We probably want to avoid having an example for every piece of functionality.

Maybe we could instead change the `tour`?

Reply:
Yeah let me extend `tour` instead!

Reply:
Done in [1a0f0c2](https://github.com/iced-rs/iced/pull/1702/commits/1a0f0c2a5368561984b5a124bb70e7ec6d900715).